Anlab.Mvc/Views/Lab/UpdateFromCompletedTests.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using Humanizer
+@model AnlabMvc.Models.Order.OrderReviewModel
+
+@{
+    ViewData["Title"] = "Confirmation";
+}
+
+@Html.Partial("_OrderDetails")
+
+<form asp-controller="Lab" asp-action="UpdateFromCompletedTests" asp-route-id="@Model.Order.Id" method="post" class="form-horizontal">
+    @Html.Hidden("confirm", true)
+    <button type="submit" class="btn btn-primary"><i class="fa fa-check" aria-hidden="true"> Confirm Completed (Update Tests from Labworks)</i></button>    
+</form>
+
+
+<div class="form-group">
+    <strong>json Data</strong>
+    <span>@Model.Order.JsonDetails</span>
+</div>
+
+@section AdditionalStyles
+{
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" />
+}
+
+@section Scripts
+{
+    <script type="text/javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
```

Comment:
@srkirkland K. This was just a copy and past of the confirmation page with a couple minor changes.